robot.py

```diff
@@ -134,7 +137,10 @@ def teleopPeriodic(self) -> None:
         else:
             self.chassis.drive_local(joystick_x, joystick_y, joystick_z)
 
-        if self.joystick.getTrigger():
+        if self.joystick.getRawButtonPressed(11):
+            self.auto_shoot = not self.auto_shoot
```

Comment:
Can we afford two buttons for this? It'd probably be a good idea to make one button enable and another disable, since there will be basically no feedback to the driver for this until a shot is (not) taken.

Reply:
I will change that because it's a good suggestion - but there will (eventually!) be feedback from the lighting system.